openassessment/xblock/static/js/spec/lms/oa_listing.js

```diff
@@ -0,0 +1,281 @@
+/**
+ Tests for course items listing.
+ **/
+
+describe("OpenAssessment.CourseItemsListingView", function() {
+
+    // Stubs
+    var view = null;
+    var rootElement = null;
+    var section = null;
+    var runtime = {
+        handlerUrl: function(el, handler) {
+            return '/' + handler;
+        }
+    };
+    window.XBlock = {
+        initializeBlock: function(el){}
+    };
+
+    var createCourseItemsListingView = function(template) {
+        loadFixtures(template);
+
+        section = $('.open-response-assessment-block');
+        rootElement = section.parent();
+        return new OpenAssessment.CourseItemsListingView(runtime, rootElement);
+    };
+
+    var stubAjax = function(success, responseData) {
+        spyOn($, 'ajax').and.returnValue(
+            $.Deferred(function(defer) {
+                if (success) {
+                    defer.resolveWith(this, [responseData]);
+                } else {
+                    defer.reject();
+                }
+           }).promise()
+        );
+    };
+
+    var testData = [{
+        "parent_name": "Vertical 1",
+        "name": "Test ORA 1",
+        "url_grade_available_responses": "/grade_available_responses_view",
+        "staff_assessment": false,
+        "parent_id": "block-v1:SomeOrg+ORA101+2017+type@vertical+block@5570454d5dc4469ca75f36dd792ee316",
+        "url_base": "/student_view",
+        "id": "block-v1:SomeOrg+ORA101+2017+type@openassessment+block@9d1af6220a4d4ecbafb22a3506effcce"
+    }, {
+        "parent_name": "Vertical 2",
+        "name": "Test ORA 2",
+        "url_grade_available_responses": "/grade_available_responses_view",
+        "staff_assessment": true,
+        "parent_id": "block-v1:SomeOrg+ORA101+2017+type@vertical+block@90b4edff50bc47d9ba037a3180c44e97",
+        "url_base": "/student_view",
+        "id": "block-v1:SomeOrg+ORA101+2017+type@openassessment+block@3ec2343a95734a87af494455f52b1141"
+    }, {
+        "parent_name": "Vertical 3",
+        "name": "Test ORA 3",
+        "url_grade_available_responses": "/grade_available_responses_view",
+        "staff_assessment": false,
+        "parent_id": "block-v1:SomeOrg+ORA101+2017+type@vertical+block@59243a76bb5e4bda9243d5ded8163e18",
+        "url_base": "/student_view",
+        "id": "block-v1:SomeOrg+ORA101+2017+type@openassessment+block@40b4edfe60bc47d9ba037a3180c44e97"
+    }];
+
```

Comment:
Created `oraCourseItems` variable: https://github.com/edx/edx-ora2/pull/979/files#diff-d629dbe5d581fab05aa0b5674c4aeef7R40

is that better?